packages/codegen-ui-react/lib/react-studio-template-renderer.ts

```diff
@@ -665,39 +668,14 @@ export abstract class ReactStudioTemplateRenderer extends StudioTemplateRenderer
   }
 
   /**
-   * case: hasVariants = true => const overrides = { ...getOverridesFromVariants(variants, props) };
-   * case: hasVariants = false => const overrides = { ...overridesProp };
+   * const overrides = mergeVariantsAndOverrides(
```

Comment:
Any reason why to keep this commented code? Its unrelated to your change but a comment helps if you know the reason.

Reply:
The comment is a hint at the rendered code is. I personally find the ast API to be somewhat tough to parse quickly, so it helps me anchor quickly into what the output of a given method is expected to be.

Reply:
Great that helps, thanks